onnxruntime/core/optimizer/identical_children_consolidation.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/graph/graph_utils.h"
+#include "identical_children_consolidation.h"
```

Comment:
Please follow the code style for header files:
[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
Checked